resources/basics.py

```diff
@@ -496,6 +496,39 @@ def qc_meta_path(data_type: str):
         return 'gs://gnomad/sample_qc/input_meta/gnomad.genomes.streamlined_metadata.2018-10-11.txt.bgz'
 
 
+# liftover data
+def get_gnomad_liftover_data_path(data_type, split=True, version=CURRENT_RELEASE) -> str:
+    """
+    Function to get paths to liftover gnomAD Tables
+    :param str data_type: One of `exomes` or `genomes`
+    :param bool split: Whether the dataset should be split
+    :param str version: One of the RELEASEs
+    :return: Path to chosen Table
+    :rtype: str
+    """
+    if version not in RELEASES:
+        return DataException("Select version as one of: {}".format(','.join(RELEASES)))
+
+    if data_type == 'exomes':
+        return f'gs://gnomad/liftover/release/2.1.1/ht/exomes/gnomad.exomes.r{version}.liftover.ht'
+    elif data_type == 'genomes':
+        return f'gs://gnomad/liftover/release/2.1.1/ht/genomes/gnomad.genomes.r{version}.liftover.ht'
```

Comment:
You might consider `f'gs://gnomad/liftover/release/2.1.1/ht/{data_type}/gnomad.{data_type}.r{version}.liftover.ht'` in lieu of the if blocks. Also, once we're satisfied with these, they should go in gnomad-public, but that can obviously wait.

Reply:
Also you'll want to put `version` in here, even though only 2.1.1 will be supported for now (or remove `version`)